test/e2e/webhook_test.go

```diff
@@ -1049,3 +1052,305 @@ var _ = Describe("Fleet's Reserved Namespace Handler webhook tests", func() {
 		})
 	})
 })
+
+var _ = Describe("Fleet's v1beta1 Resource Handler webhook tests", func() {
+	Context("fleet guard rail ClusterResourceBinding e2e tests", func() {
+		BeforeEach(func() {
+			crb := placementv1beta1.ClusterResourceBinding{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-cluster-resource-binding",
```

Comment:
Hi Arvind! If you use the same name here, it will lead to conflicts when running things in parallel.

Reply:
It's OK in the old setup as we never enabled parallelism, but for the new setup it may cease to work.